### PR TITLE
run-length-encoding: make description consistent with test cases

### LINKED ISSUE #537: [run-length-encoding] test case conflicts with description
The [description](https://github.com/exercism/x-common/blob/master/exercises/run-length-encoding/description.md) of the run-length-encoding exercise says:

> For simplicity, you can assume that the unencoded string will only contain the letters A through Z.

However, there is [one test case](https://github.com/exercism/x-common/blob/master/exercises/run-length-encoding/canonical-data.json#L47-L49) that contains spaces and lowercase letters.

I believe the test case should be changed to something like `OEJJJJJXAAPPPL`, the intent being that all of the test cases deal with strings containing only uppercase letters. I could make a PR to fix this test case, if you'd like?

### PR DESCRIPTION
**Pull request by daveyarwood**
Fixes #537.

### CODE REVIEW

File: exercises/run-length-encoding/description.md [hunk 1]
```diff
@@ -14,5 +14,11 @@ the compressed data, which makes it a lossless data compression.
...
+If the string contains any whitespace, it should be passed through unchanged:
+
+```
+"aabc dddef"  ->  "2abc 3def"
+```
+
 For simplicity, you can assume that the unencoded string will only contain
-the letters A through Z.
\ No newline at end of file
+the letters A through Z (either lower- or uppercase) and whitespace.
```
Comment (robkeim) [inline exercises/run-length-encoding/description.md]: Can you please remove the extra hyphen after lower?

**Comment by robkeim** [comment]
Looks good @daveyarwood except one minor change.

File: exercises/run-length-encoding/description.md (hunk 1 above)
Comment (daveyarwood) [inline exercises/run-length-encoding/description.md]: done!

**Comment by daveyarwood** [comment]
Change made -- I amended my previous commit to keep things tidy.

**Comment by robkeim** [comment]
Great, thanks @daveyarwood for fixing this! I'll go ahead and merge this one the Travis CI build completes.

**Pull request merged**